web/src/main/java/org/cbioportal/web/error/GlobalExceptionHandler.java

```diff
@@ -181,4 +181,10 @@
         return new ResponseEntity<>(new ErrorResponse("stableId not found: " + ex.getStableId()),
             HttpStatus.NOT_FOUND);
     }
+
+    @ExceptionHandler(ResourceDefinitionNotFoundException.class)
+    public ResponseEntity<ErrorResponse> handleResourceDefinitionNotFound(ResourceDefinitionNotFoundException ex) {
+        return new ResponseEntity<>(new ErrorResponse("Resource not found: " + ex.getResourceId()),
```

Comment:
Should ID be added to this message here?

> "Resource ID not found: " + ex.getResourceId()

instead of

> "Resource not found: " + ex.getResourceId()

I may be overthinking this but I wonder whether someone would interpret the word "Resource" alone as something related to the actual backend code instead of an error stating that a resource called `"[some_resource_id]"` could not be found for that study/patient/sample.

Reply:
I followed the same convention what's there for other exceptions(ex: StudyNotFoundException)

Reply:
Sounds fine to me - I don't really think this will be an issue anyway.